pennylane/ops/cv.py

```diff
@@ -1392,7 +1392,7 @@ def label(self, decimals=None, base_label=None, cache=None):
         return f"|{basis_string}⟩⟨{basis_string}|"
 
 
-ops = {
+__ops__ = {
```

Comment:
why do we need to make these private?

Reply:
Some pylint warnings about using `qml.ops`